style.css

```diff
@@ -0,0 +1,76 @@
+.logo {
+  height: 21px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 22px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #4c59d1;
+  margin-right: auto;
+  margin-left: 24px;
+}
+
+.topnav {
+  display: flex;
+  align-items: center;
+  padding-right: 36px;
+}
```

Comment:
### Styling

- [ ]  **Good job so far on making a good design** However, it would be better to replicate the Figma template **color**s and **fonts** for both the logo and icon as suggested in the project requirements 😎😎😎


    ### Your design
    ![image](https://user-images.githubusercontent.com/102757126/212121034-df27c768-0557-4d2f-a5ac-f778ee9c7bac.png)
    ### Figma design
    ![image](https://user-images.githubusercontent.com/102757126/212121186-805e6889-ea85-4d69-ad2e-40d0a7d3caae.png)
---
   _Remember replicating a design using CSS allows for consistency and uniformity in the appearance of a website or application. 👍👍_